libs/platform/src/lib/components/form/datetime-picker/datetime-picker/datetime-picker.component.html

```diff
@@ -0,0 +1,42 @@
+<fd-datetime-picker
+    [attr.aria-label]="ariaLabel"
+    [attr.ariaLabelledBy]="ariaLabelledBy"
+    [attr.ariaDescribedBy]="ariaDescribedBy"
+    [placeholder]="placeholder"
+    [attr.placeholder]="placeholder"
```

Comment:
Are you sure that you need both of them?
I'm not sure that you need `[attr.placeholder]="placeholder"`

Reply:
I thought this was needed for screen reader, but can't find where I read such a thing. Will remove `[attr.placeholder]`

Reply:
made the changes, please check.